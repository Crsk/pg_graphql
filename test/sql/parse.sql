select jsonb_pretty(
    (graphql.parse_query($$

        query {
          account(id: 1) {
            name
          }
        }

    $$))
)
