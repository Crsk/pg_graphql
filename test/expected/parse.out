select jsonb_pretty(
    (graphql.parse_query($$

        query {
          account(id: 1) {
            name
          }
        }

    $$))
)
                         jsonb_pretty                         
--------------------------------------------------------------
 {                                                           +
     "loc": {                                                +
         "end": {                                            +
             "line": 7,                                      +
             "column": 10                                    +
         },                                                  +
         "start": {                                          +
             "line": 3,                                      +
             "column": 9                                     +
         }                                                   +
     },                                                      +
     "kind": "Document",                                     +
     "definitions": [                                        +
         {                                                   +
             "loc": {                                        +
                 "end": {                                    +
                     "line": 7,                              +
                     "column": 10                            +
                 },                                          +
                 "start": {                                  +
                     "line": 3,                              +
                     "column": 9                             +
                 }                                           +
             },                                              +
             "kind": "OperationDefinition",                  +
             "name": null,                                   +
             "operation": "query",                           +
             "directives": null,                             +
             "selectionSet": {                               +
                 "loc": {                                    +
                     "end": {                                +
                         "line": 7,                          +
                         "column": 10                        +
                     },                                      +
                     "start": {                              +
                         "line": 3,                          +
                         "column": 15                        +
                     }                                       +
                 },                                          +
                 "kind": "SelectionSet",                     +
                 "selections": [                             +
                     {                                       +
                         "loc": {                            +
                             "end": {                        +
                                 "line": 6,                  +
                                 "column": 12                +
                             },                              +
                             "start": {                      +
                                 "line": 4,                  +
                                 "column": 11                +
                             }                               +
                         },                                  +
                         "kind": "Field",                    +
                         "name": {                           +
                             "loc": {                        +
                                 "end": {                    +
                                     "line": 4,              +
                                     "column": 18            +
                                 },                          +
                                 "start": {                  +
                                     "line": 4,              +
                                     "column": 11            +
                                 }                           +
                             },                              +
                             "kind": "Name",                 +
                             "value": "account"              +
                         },                                  +
                         "alias": null,                      +
                         "arguments": [                      +
                             {                               +
                                 "loc": {                    +
                                     "end": {                +
                                         "line": 4,          +
                                         "column": 24        +
                                     },                      +
                                     "start": {              +
                                         "line": 4,          +
                                         "column": 19        +
                                     }                       +
                                 },                          +
                                 "kind": "Argument",         +
                                 "name": {                   +
                                     "loc": {                +
                                         "end": {            +
                                             "line": 4,      +
                                             "column": 21    +
                                         },                  +
                                         "start": {          +
                                             "line": 4,      +
                                             "column": 19    +
                                         }                   +
                                     },                      +
                                     "kind": "Name",         +
                                     "value": "id"           +
                                 },                          +
                                 "value": {                  +
                                     "loc": {                +
                                         "end": {            +
                                             "line": 4,      +
                                             "column": 24    +
                                         },                  +
                                         "start": {          +
                                             "line": 4,      +
                                             "column": 23    +
                                         }                   +
                                     },                      +
                                     "kind": "IntValue",     +
                                     "value": "1"            +
                                 }                           +
                             }                               +
                         ],                                  +
                         "directives": null,                 +
                         "selectionSet": {                   +
                             "loc": {                        +
                                 "end": {                    +
                                     "line": 6,              +
                                     "column": 12            +
                                 },                          +
                                 "start": {                  +
                                     "line": 4,              +
                                     "column": 26            +
                                 }                           +
                             },                              +
                             "kind": "SelectionSet",         +
                             "selections": [                 +
                                 {                           +
                                     "loc": {                +
                                         "end": {            +
                                             "line": 5,      +
                                             "column": 17    +
                                         },                  +
                                         "start": {          +
                                             "line": 5,      +
                                             "column": 13    +
                                         }                   +
                                     },                      +
                                     "kind": "Field",        +
                                     "name": {               +
                                         "loc": {            +
                                             "end": {        +
                                                 "line": 5,  +
                                                 "column": 17+
                                             },              +
                                             "start": {      +
                                                 "line": 5,  +
                                                 "column": 13+
                                             }               +
                                         },                  +
                                         "kind": "Name",     +
                                         "value": "name"     +
                                     },                      +
                                     "alias": null,          +
                                     "arguments": null,      +
                                     "directives": null,     +
                                     "selectionSet": null    +
                                 }                           +
                             ]                               +
                         }                                   +
                     }                                       +
                 ]                                           +
             },                                              +
             "variableDefinitions": null                     +
         }                                                   +
     ]                                                       +
 }
(1 row)

